packages/react/src/primitives/shared/types.ts

```diff
@@ -179,6 +180,7 @@ export enum ComponentClassName {
   CollectionSearch = 'amplify-collection-search',
   CollectionPagination = 'amplify-collection-pagination',
   CountryCodeSelect = 'amplify-countrycodeselect',
```

Comment:
Is this no longer used?

Reply:
We are migrating our codebase away from using this class name to apply any styling, but this class needs to continue being applied to the component for any customer styling that is being applied using it.  There is a note on the breaking changes ticket to remove this class with the PR that will release the breaking changes.